docs/literate.jl

```diff
@@ -16,16 +16,24 @@ Pkg.instantiate()
 pkg_status = sprint() do io
     Pkg.status(; io=io)
 end
-manifest_status = sprint() do io
-    Pkg.status(; io=io, mode=Pkg.PKGMODE_MANIFEST)
-end
 
 using Literate: Literate
 
 const MANIFEST_OUT = joinpath(EXAMPLE, "Manifest.toml")
 mkpath(joinpath(OUTDIR, EXAMPLE))
 cp(joinpath(EXAMPLEPATH, "Manifest.toml"), joinpath(OUTDIR, MANIFEST_OUT); force=true)
 
+""" adapted from HttpCommon.jl """
+function escapeHTML(i::String)
+    # Refer to http://stackoverflow.com/a/7382028/3822752 for spec. links
+    o = replace(i, "&" => "&amp;")
+    o = replace(o, "\"" => "&quot;")
+    o = replace(o, "'" => "&#39;")
+    o = replace(o, "<" => "&lt;")
+    o = replace(o, ">" => "&gt;")
```

Comment:
In Julia >= 1.7 (IIRC) this is just
```suggestion
    o = replace(i, "&" => "&amp;", "\"" => "&quot;", "'" => "&#39;", "<" => "&lt;", ">" => "&gt;")
```

Reply:
yes, that's what I had before, but then it wouldn't be backwards-compatible anymore, and we currently claim compatibility with julia 1.3, that's why I reverted it again in 183327a

Reply:
It'd be nice to allow people to build the docs with the LTS at least, do you prefer to prevent that?

Reply:
In this package the lower bound is 1.6, and we run tests against that so might as well support in in docs as well